distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -48,8 +99,10 @@ public void beforeExam() throws Exception {
 
     @Test
     public void testCswRegistryIngest() throws Exception {
-        getServiceManager().startFeature(true, CATALOG_REGISTRY);
-        given().body(Library.getCswRegistryInsert())
+
+        given().body(Library.getCswRegistryInsert()
+                .replaceAll("urn:uuid:2014ca7f59ac46f495e32b4a67a51276",
+                        "urn:uuid:2014ca7f59ac46f495e32b4a67a51279"))
```

Comment:
Only expecting the response code of 200 isn't a great assertion - you should also inspect the contents of the response to ensure your assumptions.